dlk/python/dlk/templates/src/func/impl/arm_neon/quantized_conv2d_tiling.cpp

```diff
@@ -69,7 +69,7 @@ void pack_input_for_tiling(QUANTIZED_NOT_PACKED input[],
               + row * in_width * in_bitwidth
               + col * in_bitwidth
               + in_bit_ch;
-            output[index] |= (QUANTIZED_PACKED)bit << in_ch_low;
+            output[index] = QUANTIZED_PACKED(static_cast<QUANTIZED_PACKED::T>(output[index]) | (QUANTIZED_PACKED::T)bit << in_ch_low);
```

Comment:
Maybe we could define `operator|=` on `QuantizedPacked` as well and rewrite it somehow like
```cpp
output[index] |= QUANTIZED_PACKED(bit << in_ch_low);
```
? :thinking:

Reply:
Defined `operator|=`.